packages/ssz/src/core/deserialize.ts

```diff
@@ -152,8 +152,8 @@ function _deserializeUint(data: Buffer, type: UintType, start: number): Uint {
   if (type.byteLength > 6 && type.useNumber && uintData.equals(Buffer.alloc(type.byteLength, 255))) {
     return Infinity;
   } else {
-    const bn = new BN(uintData, 16, "le");
-    return (type.useNumber || type.byteLength <= 6) ? bn.toNumber() : bn;
+    const bn = BigInt("0x" + uintData.reverse().toString("hex"));
```

Comment:
can you use `toBigIntLE`?
also change bn variable name

Reply:
Not 100% but I think we ain't suppose to have eth2.0-utils dependency in ssz

Reply:
Cool. Ill fix. This was before @wemeetagain blessed me with the bigint-buffer library. Forgot to update here